docker/README.md

```diff
@@ -3,12 +3,24 @@
 The following docker files are split up for the queue-server and http-server for testing. The docker compose here will spin them up together along with a networked redis server. They are bare bones and will lack the full functionality of a collection profile, but should be useful for simple testing.
 
 ## To run the containers:
+On a Mac, to use the display you will need to install XQuartz and allow connections from network clients.
```

Comment:
I think "Allow connections from network clients" might be a security vulnerability that will generally get flagged by cybersecurity scans.

Reply:
This is on the money. I've fixed this to include the specific IP necessary prior to starting the container.

Reply:
Good idea!